test/generate-testcase.js

```diff
@@ -17,7 +17,7 @@ var FFX_UA = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.10; rv:38.0) Gecko/20100
 if (process.argv.length < 3) {
   console.error("Need at least a destination slug and potentially a URL (if the slug doesn't have source).");
   process.exit(0);
-  return;
+  throw "Abort";
```

Comment:
I mean, it doesn't really matter considering we're calling `process.exit` on the previous line and this is just a utility script, but return was technically illegal, so I literally wrote the shortest thing I could think of.